bigtable/google/cloud/bigtable/instance.py

```diff
@@ -481,3 +483,99 @@ def list_app_profiles(self):
         """
         resp = self._client._instance_admin_client.list_app_profiles(self.name)
         return [AppProfile.from_pb(app_profile, self) for app_profile in resp]
+
+    def get_iam_policy(self):
+        """Gets the access control policy for an instance resource.
+
+        .. code-block:: python
+
+            from google.cloud.bigtable.client import Client
+            from google.cloud.iam import Policy
+
+            client = Client(admin=True)
+            instance = client.instance('[INSTANCE_ID]')
+            policy_latest = instance.get_iam_policy()
+            print (policy_latest.owners)
+
+        :rtype: :class:`~google.cloud.iam.Policy`
+        :returns: A :class:`~google.cloud.iam.Policy` instance.
+        """
+        instance_admin_client = self._client._instance_admin_client
+        resp = instance_admin_client.get_iam_policy(resource=self.name)
+        return Policy.from_api_repr(self._to_dict_from_policy_pb(resp))
+
+    def set_iam_policy(self, policy):
+        """Sets the access control policy on an instance resource. Replaces any
+        existing policy.
+
```

Comment:
I would suggest to add a little more description about a `Policy` here
```python
        A Policy consists of a list of bindings. A binding binds a list of
        members to a role, where the members can be user accounts, Google
        groups, Google domains, and service accounts. A role is a named list
        of permissions defined by IAM.
        For more information about predefined roles currently supoprted
        by Bigtable Instance please see
        `Predefined roles
        <https://cloud.google.com/bigtable/docs/access-control#roles>`_.
        For more information about custom roles please see
        `Custom roles
        <https://cloud.google.com/bigtable/docs/access-control#custom-roles>`_.
```

Reply:
Added the detailed description.